recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -0,0 +1,53 @@
+inherit cmake pythonnative
+
+S = "${WORKDIR}/git"
+OECMAKE_SOURCEPATH = "${S}/"
+
+SECTION = "net"
+LICENSE = "BSD-2-Clause-Patent"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
+DEPENDS = "python-native python-pyyaml-native json-c readline openssl"
```

Comment:
Do we still depend on readline? I thought that was removed...

You probably want to add zmq as well.

Reply:
1) If remove `readline` from DEPENDS then Yocto prints warning
```
WARNING: prplmesh-1.0-r0 do_package_qa: QA Issue: prplmesh rdepends on readline, but it isn't a build dependency, missing readline in DEPENDS or PACKAGECONFIG? [build-deps]
```
I'll keep `readline` in RDEPENDS

2) beerocks_cli still has dependency on `readline`
https://github.com/prplfoundation/prplMesh/blob/56b858fb6921c21fc5e1549736509b4fdb5a5074/controller/src/beerocks/cli/CMakeLists.txt#L29